robot/EDA/tests/browser/settings/eda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,126 @@
+*** Settings ***
+
+Resource        robot/EDA/resources/EDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Validate Edit Mode For Accounts and Contacts, Settings
+    [tags]  unstable
+    Go To Eda Settings
+    Wait for Locator            	tabs.accountsandcontacts
+    Click on Element                tabs.accountsandcontacts
+    
+    #Go into Edit mode 
+    #Click Button                    Edit
+    Click on Element                account_types.edit
+
+    # Select Multi-Address Account Types
+    Click on Element                account_types.administrative
+
+    Wait for Locator                account_types.academic
+    Click on Element                account_types.academic
```

Comment:
You could cut the size of this test almost in half and also make it easier to read if you moved the `Wait for Locator` inside the `Click on Element` keyword.

Reply:
Will do in round 2